### PR TITLE
Add Docs diagram release versioning

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Add] Docs: Tagging and releasing versions

### CODE REVIEW

**Review by smith120bh — CHANGES_REQUESTED** [review]
I love the versioning instructions and description. But for release, let's just make this a GitHub Action. Should be pretty straightforward, and make it considerably easier for both ourselves and any contractors. See my comments on Slack.
https://clearcalcs.slack.com/archives/C06E11WFJKA/p1719379658121879?thread_ts=1719373067.808309&cid=C06E11WFJKA

**Comment by michaellisitsa** [comment]
@smith120bh as discussed separate ticket/PR will be made for automating CI workfow. Can you re-review?

File: docs/quick-start-version-control.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## GitHub Releases
+
+Once a tag has been made, a new Release can be created against this tag with relevant compiled assets.
+
+1. Parcel Cache may cause interactive diagram compiled html to not re-generate. Clear this via:
+
+```bash
+cd path/to/repository/folder
+rm -rf .parcel-cache/
```
Comment (JulianGoh5) [inline docs/quick-start-version-control.md:52]: > rm -rf

Use wisely 🤠

**Review by JulianGoh5 — APPROVED** [review]
Quick start to version control looks good, happy to send this

Earlier discussion around automation has been captured in https://app.clickup.com/t/86cvtnyj4

**Pull request merged**